bot/exts/evergreen/battleship.py

```diff
@@ -140,9 +140,11 @@ def format_grid(player: Player, emojiset: EmojiSet) -> str:
     @staticmethod
     def get_square(grid: Grid, square: str) -> Square:
         """Grabs a square from a grid with an inputted key."""
+        #changed to upper case
+        square[0].upper()
```

Comment:
String.upper is non-mutatating, meaning this change doesn't actually change the value stored in the list. You have to store this value and pass it to the function on the next line, or call upper on the next line directly.

Reply:
ind=square[0].upper()
index = ord(ind) - ord("A") 
is this ok?